test/test.sh

```diff
@@ -15,131 +15,131 @@ WHITE="\e[0;37m"
 
 
 function_message_title () {
-    echo ""
-    echo ""
-    echo ""
-    echo ""
-    echo ""
-    echo -e "${CYAN}"
-    echo -e "# | ::::::::::::::::::::::::::::::::::::::::::::: | #"
-    echo -e "# |      ${RS} $1 ${CYAN}"
-    echo -e "# | ::::::::::::::::::::::::::::::::::::::::::::: | #"
-    echo -e "${RS}"
+  echo ""
+  echo ""
+  echo ""
+  echo ""
+  echo ""
+  echo -e "${CYAN}"
+  echo -e "# | ::::::::::::::::::::::::::::::::::::::::::::: | #"
+  echo -e "# |      ${RS} $1 ${CYAN}"
+  echo -e "# | ::::::::::::::::::::::::::::::::::::::::::::: | #"
+  echo -e "${RS}"
 }
 
 
 color () {
-    # echo ""
-    #black
-    echo -e "\e[0;30m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 01 0;30m"
-    #red
-    echo -e "\e[0;31m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 02 0;31m"
-    #green
-    echo -e "\e[0;32m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 03 0;32m"
-    #yellow
-    echo -e "\e[0;33m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 04 0;33m"
-    #blue
-    echo -e "\e[0;34m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 05 0;34m"
-    #purple
-    echo -e "\e[0;35m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 06 0;35m"
-    #cyan
-    echo -e "\e[0;36m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 07 0;36m"
-    #white
-    echo -e "\e[0;37m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 08 0;37m"
-    echo ""
-    #black
-    echo -e "\e[1;30m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 09 1;30m"
-    #red
-    echo -e "\e[1;31m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 10 1;31m"
-    #green
-    echo -e "\e[1;32m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 11 1;32m"
-    #yellow
-    echo -e "\e[1;33m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 12 1;33m"
-    #blue
-    echo -e "\e[1;34m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 13 1;34m"
-    #purple
-    echo -e "\e[1;35m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 14 1;35m"
-    #cyan
-    echo -e "\e[1;36m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 15 1;36m"
-    #white
-    echo -e "\e[1;37m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 16 1;37m"
-    echo ""
-    echo -e "\e[0;30m█████\\e[0m\e[0;31m█████\\e[0m\e[0;32m█████\\e[0m\e[0;33m█████\\e[0m\e[0;34m█████\\e[0m\e[0;35m█████\\e[0m\e[0;36m█████\\e[0m\e[0;37m█████\\e[0m"
-    echo -e "\e[0m\e[1;30m█████\\e[0m\e[1;31m█████\\e[0m\e[1;32m█████\\e[0m\e[1;33m█████\\e[0m\e[1;34m█████\\e[0m\e[1;35m█████\\e[0m\e[1;36m█████\\e[0m\e[1;37m█████\\e[0m"
+  # echo ""
+  #black
+  echo -e "\e[0;30m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 01 0;30m"
```

Comment:
Could you change these to

``` bash
echo -e "\e[0;30m ███ *** AaBbCs ---  ███ \\e[0m   ---> Color 01 0;30m" # black
```

Add comments after each line, and no need for the commented `echo` at the start of function

Reply:
I did not change anything in this file except for the indentetion. But I could change your proposed suggestion

Reply:
Just intending to clean up the files in the process.